Workbooks/CosmosDb Fleet/Overview.workbook

```diff
@@ -0,0 +1,2315 @@
+                  "style": "link"
+                },
+                {
+                  "id": "f5ef2d1d-8ab4-4c8c-be77-71446912e4f0",
+                  "cellValue": "selectedTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Requests",
+                  "subTarget": "Failures",
+                  "style": "link"
+                },
+                {
+                  "id": "9ba675b6-d5b8-47ca-b7d0-acc1f60dba68",
+                  "cellValue": "selectedTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Storage",
+                  "subTarget": "Capacity",
+                  "style": "link"
+                },
+                {
+                  "id": "1fe00bb6-a177-4982-b0b7-599dfd55bede",
+                  "cellValue": "selectedTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Operations",
+                  "subTarget": "Operation",
+                  "style": "link"
+                },
+                {
+                  "id": "79497312-ed59-4485-9641-e50049a4f0fa",
+                  "cellValue": "selectedTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Throughput Pool",
+                  "subTarget": "Pool",
+                  "style": "link"
+                }
+              ]
+            },
+            "name": "links-tabs",
+            "styleSettings": {
+              "margin": "-15px 0px -15px 0px"
+            }
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "### Start editing the Overview section here"
+            },
+            "conditionalVisibility": {
+              "parameterName": "1",
+              "comparison": "isEqualTo",
+              "value": "2"
+            },
+            "name": "OverviewSectionEditStart"
+          },
+          {
+            "type": 10,
+            "content": {
+              "chartId": "workbookdb19a8d8-91af-44ea-951d-5ffa133b2ebe",
+              "version": "MetricsItem/2.0",
+              "size": 2,
+              "chartType": 0,
+              "resourceType": "microsoft.documentdb/databaseaccounts",
+              "metricScope": 0,
+              "resourceParameter": "Resources",
+              "resourceIds": [
+                "{Resources}"
+              ],
+              "timeContextFromParameter": "TimeRange",
+              "timeContext": {
+                "durationMs": 14400000
+              },
+              "metrics": [
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-TotalRequestUnits",
+                  "aggregation": 1,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50,
```

Comment:
Please remove all these split by limits (if none specified, split by limit is 1000